live-examples/js-examples/string/string-substring.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var str = 'Mozilla';
+
+console.log(str.substring(1, 2));
+// expected output: "oz"
```

Comment:
This still seems to be wrong. It should be either `(1, 3)` or `expected output: "o"`